packages/design-systems/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@tailor-platform/design-systems",
-  "version": "0.27.0",
+  "version": "0.27.1",
```

The last thing. This should be a new feature, so bump the minor instead of the patch that is for a bug fix.

```suggestion
  "version": "0.28.0",
```